.github/workflows/update-traffic-badge.yml

```diff
@@ -0,0 +1,45 @@
+name: Update traffic badge
+
+on:
+  push:            # update when you push to your fork/branch
+    branches: ["**"]
+  schedule:
+    - cron: "7 3 * * *"   # daily
```

Comment:
maybe we put:

```
    - cron: "0 0 * * *"   # daily
```

instead of 3:07 UTC time haha.

Reply:
Only thing is that it adds a commit each day, which might be annoying to have this constantly on the main branch.